src/Routing.js

```diff
@@ -122,6 +123,12 @@ export default function Routing({ appProps }) {
       allowedIf: userIsOfficerOrAdmin,
       redirect: '/',
       inAdminNavbar: true
+    },
+    {
+      Component: URLShortenerPage,
+      path: '/s',
```

Comment:
`/s` is for talking to cleezy from the internet, lets do something else like

```suggestion
      path: '/short',
```